modules/applications-create-using-cli-modify.adoc

```diff
@@ -134,6 +151,7 @@ objects.
 
 To output `new-app` artifacts to a file, edit them, then create them:
 
+[source,terminal]
 ----
 $ oc new-app https://github.com/openshift/ruby-hello-world \
     -o yaml > myapp.yaml
```

Comment:
Perhaps we can break these commands into three separate code chunks so the copy button is relevant for all of them?

Reply:
Thanks Cody for highlighting this. Resolved.